[2023-04-14T20:17:21.953+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.choose_endpoint_Odesa scheduled__2023-04-14T19:00:00+00:00 [queued]>
[2023-04-14T20:17:21.980+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.choose_endpoint_Odesa scheduled__2023-04-14T19:00:00+00:00 [queued]>
[2023-04-14T20:17:21.982+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-04-14T20:17:21.986+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-04-14T20:17:21.988+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-04-14T20:17:22.028+0000] {taskinstance.py:1309} INFO - Executing <Task(BranchPythonOperator): choose_endpoint_Odesa> on 2023-04-14 19:00:00+00:00
[2023-04-14T20:17:22.039+0000] {standard_task_runner.py:55} INFO - Started process 15527 to run task
[2023-04-14T20:17:22.045+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'weather_dag', 'choose_endpoint_Odesa', 'scheduled__2023-04-14T19:00:00+00:00', '--job-id', '598', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmp3hyfxah9']
[2023-04-14T20:17:22.047+0000] {standard_task_runner.py:83} INFO - Job 598: Subtask choose_endpoint_Odesa
[2023-04-14T20:17:22.203+0000] {task_command.py:389} INFO - Running <TaskInstance: weather_dag.choose_endpoint_Odesa scheduled__2023-04-14T19:00:00+00:00 [running]> on host ffea11fb791c
[2023-04-14T20:17:22.343+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=weather_dag
AIRFLOW_CTX_TASK_ID=choose_endpoint_Odesa
AIRFLOW_CTX_EXECUTION_DATE=2023-04-14T19:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-04-14T19:00:00+00:00
[2023-04-14T20:17:22.346+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:313: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2023-04-14T20:17:22.347+0000] {python.py:177} INFO - Done. Returned value was: retrieve_historical_weather_data_Odesa
[2023-04-14T20:17:22.348+0000] {python.py:211} INFO - Branch callable return retrieve_historical_weather_data_Odesa
[2023-04-14T20:17:22.349+0000] {skipmixin.py:155} INFO - Following branch retrieve_historical_weather_data_Odesa
[2023-04-14T20:17:22.351+0000] {skipmixin.py:211} INFO - Skipping tasks ['retrieve_weather_data_Odesa']
[2023-04-14T20:17:22.428+0000] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=choose_endpoint_Odesa, execution_date=20230414T190000, start_date=20230414T201721, end_date=20230414T201722
[2023-04-14T20:17:22.468+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-04-14T20:17:22.623+0000] {taskinstance.py:2596} INFO - 1 downstream tasks scheduled from follow-on schedule check
